newswires/client/src/Item.stories.tsx

```diff
@@ -50,11 +52,13 @@ const meta = {
 		(Story) => (
 			<EuiProvider colorMode="light">
 				<TelemetryContextProvider sendTelemetryEvent={console.log}>
-					<SearchContextProvider>
-						<div style={{ maxWidth: '800px', margin: '0 auto' }}>
-							<Story />
-						</div>
-					</SearchContextProvider>
+					<UserSettingsContextProvider>
```

Comment:
I might be missing something here, but I'm curious as to why this provider has been introduced and not used?

Reply:
We wrote the the `useUserSettings` hook so that it throws an error at runtime if it's called outside of a `UserSettingsContextProvider` instance. In this case, the `Item` component has a child that uses the `useUserSettings` hook: https://github.com/guardian/newswires/blob/a9702cd7d0ca1092d2fcfa9e787565d98e4f502a/newswires/client/src/ToolsConnection.tsx#L181

I think the broader pattern comes from this post: https://kentcdodds.com/blog/how-to-use-react-context-effectively#typescript